src/widgets/category/title/search-title-template.tpl

```diff
@@ -39,8 +39,8 @@
         </span>
 
         <% if (!isSourceType) { %>
-          <span class="CDB-Text CDB-Size-small u-lSpace--s" style="color: <%- sourceColor %>;">
-            <i class="CDB-IconFont CDB-IconFont-ray"></i>
+          <span class="CDB-Text CDB-Size-small u-lSpace--s u-flex" style="color: <%- sourceColor %>;">
+            <i class="CDB-IconFont CDB-IconFont-ray CDB-Size-small"></i>
```

Comment:
Does it make sense using `CDB-Size-small` without `CDB-Text` in that same element?

Reply:
By [watching the classes](https://github.com/CartoDB/CartoAssets/blob/master/src/scss/cdb-components/_typography.scss), it makes sense (I mean, it works), but maybe semantically it doesn't make sense

Reply:
I've had unintended consequences before using both classes separately then together, I mean they are not really utility classes but they affect each other's behaviour :/

That's why I would be concerned

Reply:
I tried to change it and I realized one thing, if I use CDB-Text it changes the font-family, so the icon disappear :joy:

Maybe it's just a naming problem? And it should be `CDB-Font` instead of `CDB-Text` so we have `CDB-Font / CDB-Iconfont` which changes `font-family` and then `CDB-Size`?

Reply:
Ha! That's what I'm talking about, if it looks ok with the current changes, then 👍